students/sangwoong/users/views.py

```diff
@@ -1,3 +1,49 @@
 from django.shortcuts import render
 
 # Create your views here.
+import json, re
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class SignUpView(View):
+  
+  def post(self, request) :
+   try:
+    data          = json.loads(request.body)
+    last_name     = data["last_name"]
+    first_name    = data["first_name"]
+    email         = data["email"]
+    account       = data["account"]
+    pw            = data["password"]
+    phone_number  = data["phone_number"]
+    birth         = data["date_of_birth"]
```

Comment:
변수명은 최대한 줄이지 말고 전부 적어주세요.
```suggestion
    data          = json.loads(request.body)
    last_name     = data["last_name"]
    first_name    = data["first_name"]
    email         = data["email"]
    account       = data["account"]
    password     = data["password"]
    phone_number  = data["phone_number"]
    date_of_birth = data["date_of_birth"]
```